src/lib/index.ts

```diff
@@ -1,7 +1,38 @@
 import fetch from 'node-fetch';
 import path from 'path';
 import execSh from 'exec-sh';
-import { specmaticJarPathLocal } from '../config';
+import { specmaticJarPathLocal, specmatic } from '../config';
+import fs from 'fs';
+import { check } from 'yargs';
+
+type Environment = Record<string, string>
+
+export const setSpecmaticEnvironement = (environmentName: string, environment: Environment) => {
+  console.log("Reading the file from -> ", path.resolve(specmatic))
+  let file = null
+
+  try {
+    file = require(path.resolve(specmatic))
+    for (var x in environment) { file.environments[environmentName].variables[x] = environment[x] }
+    fs.writeFileSync(path.resolve(specmatic), JSON.stringify(file, null, 2))
+    return checkSpecmaticEnvironment(environmentName, environment)
```

Comment:
Why are we checking here. The purpose of the method is to setSpecmaticEnvironment. Checking if it is set properly or not is the concern of the test for this method.

Reply:
It is just the name for a function, in that function I am reading the file again and checking whether the right variables are updated or not based on which that check function returns true or false and this returned boolean value, in turn, gets returned from the main function setSpecmaticEnvironment

Reply:
Made the changes the checking is being done inside the unit test